app/views/admin/tickets/show.html.haml

```diff
@@ -19,6 +19,7 @@
         %th E-Mail
         %th Affiliation
         %th Paid
+        %th Unpaid
```

Comment:
I think that we don't need an unpaid column. I would render in the paid one how many tickets out of the total are paid.

I mean, if a user bought 3 tickets and paid 2 render: `2/3`